mm/highmem.c

```diff
@@ -164,6 +164,10 @@ struct page *kmap_to_page(void *vaddr)
 }
 EXPORT_SYMBOL(kmap_to_page);
 
+#ifdef CONFIG_LIB
```

Comment:
if this is just a stub (glue) code, then it is nice to put it in arch/lib/glue.c or somewhere in arch/lib.